src/common/combocmn.cpp

```diff
@@ -2284,6 +2285,9 @@ void wxComboCtrlBase::ShowPopup()
 
     SetFocus();
 
+    int displayIdx = wxDisplay::GetFromWindow(this);
+    wxRect displayRect = wxDisplay(displayIdx != wxNOT_FOUND ? displayIdx : 0u).GetGeometry();
```

Comment:
Apparently there's already a convenient ctor for `wxDisplay` taking a `wxWindow*`, which could be used here and elsewhere later (I was going to suggest it as perhaps being useful to have but I was in v3.0 which didn't have it yet).

Reply:
Thanks, somehow I missed this.